compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -33,22 +33,34 @@
 
 namespace mlir::iree_compiler::AMDAIE {
 
+/// Command line option for selecting the lowering pipeline to use to generate
+/// AIE instructions.
+static llvm::cl::opt<LowerToAIEPassPipeline> clUseLowerToAIEPipeline(
+    "iree-amdaie-use-lower-to-aie-pipeline",
+    llvm::cl::desc("Pick the lowering pipeline to use"),
```

Comment:
This description should be more concrete and different from the one below.

Reply:
Added a more descriptive comment to the one below as that one had an unrelated comment.